packages/zql/src/query/abstract-query.ts

```diff
@@ -59,8 +58,7 @@ export abstract class AbstractQuery<
   >
   implements
     Query<TTable, TSchema, TReturn>,
-    QueryInternals<TTable, TSchema, TReturn>,
-    ToQuery<TTable, TSchema, TReturn, unknown>
```

Comment:
Query builders no longer have to implement `ToQuery` (and don't have a `query.toQuery(ctx)` fn)